mobile/examples/whisper/azure/android/readme.md

```diff
@@ -0,0 +1,39 @@
+# Android Speech Recognition Example with Azure Custom Op
+
+This example shows how to use ORT to do speech recognition using the [Whisper](https://github.com/openai/whisper) model and calls the OpenAI Whisper endpoint using the Azure custom op. You MUST update the code to specify a valid OpenAI auth token. See [here](#prepare-an-openai-auth-token) for details.
+
+The application lets the user make an audio recording, then recognizes the speech from that recording and displays a transcript.
+
+Example App Screenshot - A Successful result returned from OpenAI endpoint:
+
+<img width=25% src="images/Img_successful_endpoint_res.png" alt="App Screenshot" />
+
+## Set up
+
+### Prerequisites
+
+See the general prerequisites [here](../../../../README.md#General-Prerequisites).
+
+Additionally, you will need to be able to record audio, either on an emulator or a device.
```

Comment:
nit: we include pre-recorded audio don't we and recording your own audio is optional?

Reply:
yea, updated in both whisper samples.